packages/forms/test/form_control_spec.ts

```diff
@@ -45,6 +45,12 @@ describe('FormControl', () => {
       expect(c.status).toBe('DISABLED');
     });
 
+    it('should honor disabled when value prop is missing', () => {
```

Comment:
I'd propose to phrase it similar to the last test in the "boxed values" group:
```suggestion
    it('should not treat objects as boxed values when `disabled` field is present, but `value` is missing', () => {
```

Reply:
I agree with the suggestions, added changes. It looks like all checks passed